lib/active_encode/technical_metadata.rb

```diff
@@ -24,5 +22,12 @@ module TechnicalMetadata
       attr_accessor :audio_bitrate
       attr_accessor :video_bitrate
     end
+
+    def assign_tech_metadata metadata
```

Comment:
I still think that this isn't necessary or should be converted into a more holistic mass assignment but I could let that go for now.  It wouldn't be hard to deprecate and change this in the future if we want so I'm fine with it for now.